lib/bundler/inline.rb

```
@@ -27,7 +27,9 @@
 #
 #          puts Pod::VERSION # => "0.34.4"
 #
-def gemfile(install = false, &gemfile)
+def gemfile(install = false, options = {}, &gemfile)
+  ui = options[:ui]
```

should raise if there are any unknown options passed